README.md

```diff
@@ -42,6 +42,16 @@ Run the application on `http://localhost:3000`:
 bin/dev
 ```
 
+### BigQuery
+
+Edit `.env.development.local` and add a BigQuery key if you want to use BigQuery locally.
```

Comment:
Doesn't actually exist yet as we don't have anything to put in it, but added the gem anyway.